backend/api/v1/database_service.go

```diff
@@ -272,32 +275,6 @@ func filterProjectDatabasesV2(ctx context.Context, s *store.Store, iamManager *i
 		}
 	}
 
-	for _, binding := range policy.Bindings {
```

Comment:
Looks like duplicate logic, and we can break the loop to make it more efficient

Reply:
not duplicate. projectQuerier and projectExporter are handled differently compared to other roles because of CEL.

Reply:
Let me update this in next PR

Reply:
In the old code, in the first loop,

```
if binding.Role == api.ProjectQuerier || binding.Role == api.ProjectExporter {
  continue
}
// the role is not ProjectQuerier or ProjectExporter
// then check if the role contains the required permission
```

in the second loop,

```
if binding.Role != api.ProjectQuerier && binding.Role != api.ProjectExporter {
  continue
}
// the role is ProjectQuerier or ProjectExporter
// then check if the role contains the required permission
```

So I think we don't need to check what the role is.